Project.toml

```
@@ -13,7 +13,7 @@ MPFR_jll = "3a97d323-0669-5f0c-9066-3539efd106a3"
 [compat]
 BinaryProvider = "0.4, 0.5"
 julia = "1"
-FLINT_jll = "0.0.2"
+FLINT_jll = "^2.6.0"
```

Note that this is equivalent to this (I am fine with either one)
```suggestion
FLINT_jll = "2.6.0"
```

---

Yes, but I prefer the implicit "this is not equality, there is a range" provided by the caret.